### PR TITLE
Update to Kotlin 1.7.10

### PR DESCRIPTION
**Pull request by ZacSweers**

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -23,7 +23,8 @@ versionsPlugin = "0.42.0"
 dependencyAnalysis = { id = "com.autonomousapps.dependency-analysis", version.ref = "dependencyAnalysisPlugin" }
 detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "detekt" }
 dokka = { id = "org.jetbrains.dokka", version.ref = "dokka" }
-ksp = { id = "com.google.devtools.ksp", version.ref = "ksp" }
+# Here to trigger Renovate updates
```
Comment (ZacSweers) [inline gradle/libs.versions.toml:26]:
```suggestion
ksp = { id = "com.google.devtools.ksp", version.ref = "ksp" }
# Here to trigger Renovate updates
```

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -24,6 +24,8 @@ dependencyAnalysis = { id = "com.autonomousapps.dependency-analysis", version.re
 detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "detekt" }
 dokka = { id = "org.jetbrains.dokka", version.ref = "dokka" }
 ksp = { id = "com.google.devtools.ksp", version.ref = "ksp" }
+# Here to trigger Renovate updates
+kotlinksp = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
```
Comment (seankim-android) [inline gradle/libs.versions.toml:28]: Questions:

1. I'm curious why we're naming the jvm artifact as ksp.
2. kolinksp is all lowercase, and not kotlin-ksp. Is this intentional?

**Review by seankim-android — APPROVED** [review]
LGTM. Left questions for my understanding.

File: gradle/libs.versions.toml (hunk 2 above)
Comment (ZacSweers) [reply gradle/libs.versions.toml:28]: derp, this should be kotlin.jvm

Comment (ZacSweers) [reply gradle/libs.versions.toml:28]:
```suggestion
kotlin.jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
```

Comment (seankim-android) [reply gradle/libs.versions.toml]: do we want dot in the name? Perhaps kotlinJvm or kotlin-jvm?

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -24,6 +24,8 @@ dependencyAnalysis = { id = "com.autonomousapps.dependency-analysis", version.re
 detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "detekt" }
 dokka = { id = "org.jetbrains.dokka", version.ref = "dokka" }
 ksp = { id = "com.google.devtools.ksp", version.ref = "ksp" }
+# Here to trigger Renovate updates
+kotlin.jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
```
Comment (ZacSweers) [inline gradle/libs.versions.toml:28]:
```suggestion
kotlin-jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
```

**Comment by seankim-android** [comment]
👍 Looks great!

**Pull request merged**